images/dptp-3312-hypershift-leaks-cleaner/Dockerfile

```
@@ -1,7 +1,14 @@
+FROM registry.ci.openshift.org/ci/hypershift-cli:latest AS hypershift
+FROM registry.ci.openshift.org/ocp/4.14:cli AS oc
+
 FROM quay.io/centos/centos:stream8
 LABEL maintainer="bechen@redhat.com"
 
 RUN yum install -y unzip jq
+
+COPY --from=hypershift /usr/bin/hypershift /usr/bin/hypershift
```

With https://github.com/openshift/release/pull/42025

```suggestion
COPY hypershift /usr/bin/hypershift
```